randovania/games/prime1/logic_database/Phazon Mines.txt

```diff
@@ -402,24 +402,91 @@ Extra - aabb: [-36.12424, -29.465834, -4.4074636, 29.15716, 66.50636, 24.451797]
       All of the following:
           After Mine Security Station Unlock Doors
           Any of the following:
-              Combat (Intermediate) or Normal Damage ≥ 125
-              Shoot Ice Beam and Shoot Wave Beam
+              # Avoid Pirates
+              Combat (Expert) or Normal Damage ≥ 199
+              All of the following:
+                  # Fight Pirates
+                  Shoot Ice Beam and Shoot Wave Beam
+                  Any of the following:
+                      Combat (Advanced) or Normal Damage ≥ 149
+                      All of the following:
+                          Charge Beam
+                          Any of the following:
+                              Combat (Intermediate) or Normal Damage ≥ 99
+                              Combat (Beginner) and Normal Damage ≥ 49
```

Comment:
The problem with this is that you are logically requiring the player have the requirements to deal with both the 1st pass fight and 2nd pass pirates on the first pass.

Reply:
I'm uncertain how you'd write it to reflect that. To leave on the first pass, `After Mine Security Station Unlock Doors` needs to have happened already, but that same event is what checks if you are on the second pass. Could you use another event to make this work?

Reply:
On 2nd thought, it's actually fine because the unlock doors event lets you take a trivial shortcut around this logic.